### PR TITLE
Search Feature v1

### PR DESCRIPTION
**Pull request by guilleasz**
We made a basic forum search engine. It searches full words. We also made a regEx search, if you want we can make another pull request for it (is less performant but better user-experience)

### CODE REVIEW

File: config/defaults.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
   "publicPort": 3000,
   "mongoUrl": "mongodb://localhost/DemocracyOS-dev",
   "mongoUsersUrl": "",
-  "jwtSecret": "Generate a secret token and paste it here.",
+  "jwtSecret": "thisisthemostawesomesecrettokenevernotseenthenotispurposedly",
```
Comment (mjlescano) [inline config/defaults.json]: Be careful! You are modifying the default configuration file. You should put your local config values on `config/development.json`, more info: http://docs.democracyos.org/configuration.html

File: lib/stores/store/store.js [hunk 2]
```diff
@@ -16,6 +16,10 @@ export default class Store {
       url: null,
       items: []
     }
+    this._findForumSearchCache = {
```
Comment (mjlescano) [inline lib/stores/store/store.js]: This cache is only needed by `lib/stores/forum-store/forum-store.js`, so it should be on that file, to not pollute all the stores.

**Comment by mjlescano** [comment]
Nice! I added some comments before we can merge it. Also, make sure to indent everything with `2 spaces`.

**Comment by FacuVelasco** [comment]
thanks mati, we update the code to match your comments, will continue working in the Full-text search!

**Comment by mjlescano** [comment]
Already rebased and merged here: #1387 

Thanks a lot @guilleasz, @FacuVelasco, @atralice, @solanoepalacio, @manuelsalomon ❤️

**Pull request closed without merging**